urbit/docker-compose.yml

```diff
@@ -6,8 +6,9 @@ services:
       APP_PORT: 8090
 
   manager:
-    image: mopfelwinrux/urbit-umbrel:v0.3.0@sha256:09ee4550be66894cc2b763d5cff9a4ed15b13a3c2a298519a857c6542069d8c7
+    image: mopfelwinrux/urbit-umbrel:v0.4.1@sha256:083db8cdb2109f29c65e186b47eeecdacd5d8021ce1c4593cf8a9f945f646171
     ports:
+      - "${APP_URBIT_PORT}:8090"
```

Comment:
@mopfel-winrux Ok, no worries re `app_proxy` service. This line will also need to be dropped and then I'll give this a test :)

Reply:
Done!